views/swap/index.tsx

```diff
@@ -74,14 +90,15 @@ const Swap: FC = () => {
             mb="l"
           >
             <Button
-              bg="container"
-              opacity="0.4"
-              color="onSurface"
               px="l"
               py="s"
-              borderRadius="xs"
-              variant="tonal"
               fontSize="s"
+              opacity="0.4"
+              bg="container"
```

Comment:
Add condition to change  button text color and bg when 'from' and 'to' coins exist.